deploy/templates/base/manager_image_patch.yaml

```diff
@@ -9,8 +9,8 @@ spec:
     spec:
       containers:
       - name: devworkspace-controller
-        image: ${IMG}
+        image: quay.io/devfile/devworkspace-controller:next
```

Comment:
Was this one an accident as well? I assume we still want the templates

Reply:
true, probably I just did ctrl+c when make generate as Angel assumed. I'll revert